mobile/salto/src/routes/settings/SettingsModal.svelte

```diff
@@ -0,0 +1,101 @@
+<script lang="ts">
+  export let selectedCurrency: string;
+  export let fiatCurrencies: string[];
+  export let onClose: () => void;
```

Comment:
Use the props syntax you can find in other components.
And store the selected currency in a global store. You can find those in `mobile/salto/src/stores.ts`.
It makes more sense for the types of values we are going to be working with in `Settings`